.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```

Comment:
You should add this in your global gitignore, it's specific to your configuration :wink: